dataduct/steps/transform.py

```diff
@@ -99,7 +99,7 @@ def __init__(self,
         self.create_pipeline_object(
             object_class=ShellCommandActivity,
             input_node=input_nodes,
-            output_node=base_output_node,
```

Comment:
This should not change as the self._output contains a dictionary in the multi output case and we want the activity to write to the single S3 node with multiple directories that are later referenced in the pipeline. 

If you look at line94, the self_output in that case is a dictionary which should not be passed to ShellCommandAcvitity. I haven't had a chance to document all the steps properly as I was working on the core features for version 0.2